docs/using/web.md

```diff
@@ -55,6 +55,10 @@ To view your file as raw `html`:
 
     http://localhost:8080/test.html
 
+You may have noticed that we just created both `test.md` and `test/`. In urbit, there is no distinction between a directory and a file. So, if you're adding files directly from unix, simply `mkdir test` and urbit will treat the file and directory as the same entity. 
+
+You can use this feature to change your homepage at `your-urbit.urbit.org`. By default, this page will display some info about your urbit, but you can create the file `/home/web.md` to customize this page to your heart's content.
```

Comment:
This is great!  I had been meaning to put this in.  

`/home/web.md` actually exists by default, so could be worth rephrasing to that end.  

When we were talking earlier, I think I sort of lost the thread when I was trying to get at this exact point.  That you can customize the homepage, and pretty easily turn it into a personal talk feed, static site, or whatever.  Even the identity component can be customized.  I had intended to make this one of the children of `/docs/using/web/` (like http://urbit.org/docs/using/web/static/) but never got around to it.